internal/cli/importcmd/yaml.go

```diff
@@ -43,44 +44,77 @@ type TenantConfig struct {
 	} `yaml:"roles"`
 }
 
-func (t *TenantConfig) getYAML(yamlData string, config string) *TenantConfig {
-	type StructA struct {
-		A string `yaml:"a"`
+func ParseYAML(yamlPath string, config *Config) *TenantConfig {
```

Comment:
Could you please return the errors, instead of printing them?

Reply:
So it could return a tuple of TenantConfig, error

Reply:
Thanks, missed this in trying to get something out the door. Let me know if there's any other feedback! 🙇